go-lang/utils.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import "math/big"
+
+func ToBigBase(value *big.Int, base uint64) (res [](uint64)) {
+	toBigBaseImpl(value, base, &res)
+	return
+}
+
+func toBigBaseImpl(value *big.Int, base uint64, res *[](uint64)) {
```

Comment:
To be more in line with the standard libs I would suggest an approach in which the slice is returned and not referenced to. Does anything speak against that?